docs/examples/Example_data_fusion_from_sensors.py

```diff
@@ -0,0 +1,463 @@
+# of multi-target tracking and we compare the performances
+# of separate filters, an unscented Kalamn filter (KF),
+# an extended Kalman filter (EKF) and a particle filter (PF).
+#
+# The example layout is as follows:
+# 1) define the targets trajectories and the sensors
+# specifics for collecting the measurements;
+# 2) define the various filter components and build the trackers;
+# 3) run the measurement fusion algorithm and run the trackers;
+# 4) plot the tracks and the track performances using the metric
+# manager tool available.
+#
+
+# %%
+# 1) Define the targets trajectories and the sensors collecting the measurements;
+# -------------------------------------------------------------------------------
+# Let's define the targets trajectories, assuming a simple case
+# of a straight movement and using the same specifics for both
+# sensors. We consider a two :class:`~.RadarBearingRange` class radars
+# collecting the detections of the targets, in the same cartesian space.
+# The first radar is placed onto a :class:`~.FixedPlatform` while the
+# second is on a :class:`~.MovingPlatform`.
+# The targets follow a straight line trajectory for simplicity.
+# For the targets we instantiate the origins and a transition model
+# with :class:`~.ConstantVelocity` equal to 0.
+#
+
+# General imports
+import numpy as np
+import datetime
+from datetime import datetime
+from datetime import timedelta
+
+# Stone Soup general imports
+from stonesoup.models.transition.linear import CombinedLinearGaussianTransitionModel, \
+    ConstantVelocity
+from stonesoup.types.state import GaussianState
+from stonesoup.types.array import CovarianceMatrix
+from stonesoup.simulator.simple import MultiTargetGroundTruthSimulator
+
+# Simulation parameters setup
+start_time = datetime(2023, 8, 1, 10,0,0) # For simplicity fix a date, or datetime.now()
+number_of_steps = 50  # Number of timestep for the simulation
+np.random.seed(1908)  # Random seed for reproducibility
+
+# Specify the ground truth transition model
+gnd_transition_model = CombinedLinearGaussianTransitionModel([
+    ConstantVelocity(0.00), ConstantVelocity(0.00)])
+
+# Instantiate the target transition model 2D case,
+# insert some noise for the prediction in particle
+# filters
+transition_model = CombinedLinearGaussianTransitionModel([
+    ConstantVelocity(0.1), ConstantVelocity(0.1)])
+
+# Define the initial target state
+initial_target_state_1 = GaussianState([25, 1, 50, -0.5 ],
+                                     np.diag([1, 0.1, 1, 0.1]) ** 2,
+                                     timestamp=start_time)
+# Define the initial target state
+initial_target_state_2 = GaussianState([25, 1, -50, 0.5],
+                                     np.diag([1, 0.1, 1, 0.1]) ** 2,
+                                     timestamp=start_time)
+
+# Create a ground truth simulator, specify the number of initial targets as
+# 0 so no new targets will be created a side from the two provided
```

Comment:
```suggestion
# Create a ground truth simulator and specify the number of initial targets as
# 0, so that no new targets will be created aside from the two provided
```